app/models/issue.rb

```diff
@@ -0,0 +1,99 @@
+# == Schema Information
+#
+# Table name: issues
+#
+#  id               :integer          not null, primary key
+#  reportable_type  :string           not null
+#  reportable_id    :integer          not null
+#  reported_user_id :integer
```

Comment:
Shouldn't this be `not null` here? Is it ever possible to have a report that doesn't map to a user?

Reply:
Anonymous notes (and if added in future, anonymous note comments).